libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -922,6 +1025,70 @@ string YulUtilFunctions::updateStorageValueFunction(Type const& _type, boost::op
 	});
 }
 
+string YulUtilFunctions::updateMemoryValueFunction(Type const& _type, bool _pad)
```

Comment:
Please remove the `_pad` argument.

Reply:
Won't we need it later when we handle byte arrays?

Reply:
I think this case is way too special to warrant carrying this bool all the way through.